gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -136,12 +136,12 @@ const (
 )
 
 // AddPackage adds a package with given fileset.
-func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) error {
+func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 	creator := msg.Creator
 	pkgPath := msg.Package.Path
 	memPkg := msg.Package
 	deposit := msg.Deposit
-	store := vm.getGnoStore(ctx)
+	gnostore := vm.getGnoStore(ctx)
```

Comment:
Its intention is to differentiate the name from the imported `tm2/pkg/store` package. Otherwise, there would be a name conflict, and the code wouldn't compile.